elk.yaml

```diff
@@ -0,0 +1,25 @@
+name: elk
+description: |
+       Simple ELK stack deployment: it installs ES, Kibana and lostash as a child of Apache Tomcat 8
+services:
+- type: org.apache.brooklyn.entity.nosql.elasticsearch.ElasticSearchNode
+  id: es
+  name: elasticsearch
+  brooklyn.config:
+    install.version: 1.4.4
+- type: kibana-standalone
```

Comment:
you need to add this [catalog.bom](http://catalog.brooklyn.io/#blueprint/andreaturli/brooklyn-kibana)